pvlib/pvsystem.py

```diff
@@ -1659,12 +1660,23 @@ def sapm(effective_irradiance, temp_cell, module):
     temperature.sapm_module
     '''
 
-    T0 = 25
+    # TODO: someday, change temp_ref and irrad_ref to reference_temperature and
+    # reference_irradiance and expose
+    temp_ref = 25
+    irrad_ref = 1000
+    # TODO: remove this warning in v0.8 after deprecation period for change in
+    # effective irradiance units, made in v0.7
+    if np.all(effective_irradiance) < 2.0:
+        import warnings
+        warnings.warn('All effective_irradiance inputs to SAPM are low.'
+                      ' Units changed in v0.7 from suns to W/m2',
+                      pvlibDeprecationWarning)
```

Comment:
I'm not sure if a deprecation warning is the right class of warning if the behavior is suddenly changing. Maybe `RuntimeWarning`?